openpype/hosts/max/plugins/publish/collect_render.py

```diff
@@ -35,6 +35,8 @@ def process(self, instance):
         files_by_aov.update(aovs)
 
         camera = rt.viewport.GetCamera()
+        if instance.data.get("members"):
+            camera = instance.data["members"][-1]
```

Comment:
Is there something currently ensuring that the contained member is actually a camera? If not, we should probably prepare for that.

Reply:
we can do a check to get the list of object in instance.data["members"] which is camera. 
Some code similar to this : 
```
camera_list = [member for member in instance.data["members"] if rt.ClassOf(member) == rt.Camera.Classes]
```